src/main/java/vazkii/patchouli/api/ICommandProcessor.java

```diff
@@ -0,0 +1,11 @@
+package vazkii.patchouli.api;
+
+import vazkii.patchouli.client.book.text.SpanState;
```

Comment:
?? a proper api interface needs to be made instead of exposing internals like this

Reply:
ok will do that was a mistake